cql3/statements/create_keyspace_statement.cc

```diff
@@ -92,11 +92,12 @@ void create_keyspace_statement::validate(query_processor& qp, const service::cli
 future<std::tuple<::shared_ptr<cql_transport::event::schema_change>, std::vector<mutation>, cql3::cql_warnings_vec>> create_keyspace_statement::prepare_schema_mutations(query_processor& qp, api::timestamp_type ts) const {
     using namespace cql_transport;
     const auto& tm = *qp.proxy().get_token_metadata_ptr();
+    const auto& feat = qp.proxy().features();
     ::shared_ptr<event::schema_change> ret;
     std::vector<mutation> m;
 
     try {
-        m = service::prepare_new_keyspace_announcement(qp.db().real_database(), _attrs->as_ks_metadata(_name, tm), ts);
+        m = service::prepare_new_keyspace_announcement(qp.db().real_database(), _attrs->as_ks_metadata(_name, tm, feat), ts);
```

Comment:
Theological modularity note: feature_service is anti-modular, since it has fan-in from many modules (anything that changes over time) and fan-out to many modules (anything that depends on a feature).

We should have broken in down at high level and just sent individual features to anyone that needs them. So main.cc sends the tablets feature to query_processor, not the whole thing.

This is just for consideration, I don't expect any action.